.github/workflows/solvcon_install.yml

```diff
@@ -3,7 +3,23 @@ on:
   push:
   pull_request:
 jobs:
-  build:
+  unittest:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
```

Comment:
Would it be proper to upgrade to 20.04 with this PR?  I am fine keeping 18.04 or not.

Reply:
Sure!

Reply:
Let me use `ubuntu-latest` instead of 18.04, so we can always try the latest one. It's always proper to run the unittest with the latest ubuntu because the framework is bash, and bash is supposed to work across distributions.

Reply:
I think it would be much better to spell out `20.04`.  Explicit is better than implicit.  I was surprised by the change of CI results a couple of times with the use of similar latest labels.